src/calculateAverage.js

```diff
@@ -37,7 +37,36 @@ function applyCalculateAverage() {
    * @param {Object} options - optional
    */
   [].__proto__.calculateAverage = function(options) {
-    // write code here
+    if (options !== null) {
```

Comment:
instead of write all code inside big if will be better to opposite condition and just return undefined
Also, you do not cover cases with string, array, number etc.